### PR TITLE
Issue in deletion of vlan properties -b- #40

### PR DESCRIPTION
**Pull request by vivek-metricdust**
backend fix [#40](https://github.com/STORDIS/orca_backend/issues/40)
added support to config description as "" (null string) as it is acceptable by device for vlan

### CODE REVIEW

File: network/vlan.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def vlan_config(request):
                     device_ip,
                     vlan_name,
                     enabled=req_data.get("enabled", None),
-                    descr=req_data.get("description", ""),
+                    descr=req_data.get("description", None),
                     mtu=req_data.get("mtu", ""),
```
Comment (kamalkrbh) [inline network/vlan.py:90]: @vivek-metricdust , Better initialise the paralegals with None Instead of empty value

File: network/vlan.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def remove_vlan_ip_address(request):
             )
         vlan_ip_addr = req_data.get("vlan_ip_addr", None)
```
Comment (kamalkrbh) [inline network/vlan.py:167]: @vivek-metricdust ,  unused line.

File: network/vlan.py (hunk 1 above)
Comment (vivek-metricdust) [reply network/vlan.py]: changed in [8ae13d5](https://github.com/STORDIS/orca_backend/pull/50/commits/8ae13d540024c48283fe9584b1fc165bf7455e8c)

File: network/vlan.py (hunk 2 above)
Comment (vivek-metricdust) [reply network/vlan.py]: removed in [8ae13d5](https://github.com/STORDIS/orca_backend/pull/50/commits/8ae13d540024c48283fe9584b1fc165bf7455e8c)

**Comment by vivek-metricdust** [comment]
PR updated with support to delete sag_ip and also to add sag_ip as array

**Pull request merged**